motor_driver.c

```diff
@@ -127,19 +145,65 @@ void motor_drv_disable_motor(enum stepper_e stepper)
   _motor_drv_set_val(stepper, CTRL, ENABLE_IDX, ENABLE_MASK, 0);
 }
 
-void motor_drv_init()
+void motor_drv_inverse_dir(enum stepper_e stepper, bool reversed)
 {
-  /* Configure reset pin as output */
-  MOTOR_RESET_DDR |= ((1 << MOTOR_RESET_PIN) | (1 << MOTOR_RESET_LINE_DRIVER_PIN));
+  if (reversed) {
+    _motor_drv_set_val(stepper, CTRL, RDIR_IDX, RDIR_MASK, 1);
+  } else {
+    _motor_drv_set_val(stepper, CTRL, RDIR_IDX, RDIR_MASK, 1);
```

Comment:
I'm pretty sure this will do the same thing no matter what reversed value you pass?